MAUI/ListView/swiping.md

```diff
@@ -0,0 +1,618 @@
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void ListView_Swiping(object sender, SwipingEventArgs e)
+{
+   if (e.Index == 1 && e.OffSet > 70)
+       e.Handled = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### SwipeEnded Event
+
+The `SfListView.SwipeEnded` event is fired when completing the swipe action. This event is triggered with `SwipeEndedEventArgs`.
+
+The `SwipeEnded` event provides the following properties in their arguments: 
+
+ * `Index`: Defines the swiping item index.
+ * `DataItem`: Defines the underlying data associated with the swiped item as its arguments. 
+ * `Direction`: Defines the swipe direction of the swiped item.
+ * `OffSet`: Defines the current swipe offset of the item being swiped.
+ 
+The `SwipeEnded` event is used for the following use cases:
+ 
+ * To insert the data or edit the data after swiping.
+ * To delete the item from view after swiping it to certain extent.
+ * To reset the swipe view automatically for the swiped item.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfListView x:Name="listView" ItemsSource="{Binding InboxInfo}" AllowSwiping="True"
+                       SwipeEnded="ListView_SwipeEnded" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+listView.SwipeEnded += ListView_SwipeEnded;
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void ListView_SwipeEnded(object sender, SwipeEndedEventArgs e)
+{
+  if (e.Offset > 70)
+      listView.ResetSwipeItem();
+}
+{% endhighlight %}
+{% endtabs %}
+
+### SwipeReset Event
+
+
+The `SwipeReset` event provides the following properties in their arguments:
+
+ * `Index`: Defines the swiping item index.
+ * `DataItem`: Defines the underlying data associated with the swiped item as its arguments. 
+ * `OffSet`: Defines the current swipe offset of the item being swiped.
+ 
+The `SwipeReset` event is used for the following use case:
+
+ * To skip the reset operation for a swiped item.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfListView x:Name="listView" ItemsSource="{Binding InboxInfo}" AllowSwiping="True"
+                       SwipeReset="listView_SwipeReset" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+listView.SwipeReset += ListView_SwipeReset;
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void ListView_SwipeReset(object sender, ResetSwipeEventArgs e)
+{
+  if (e.Index == 1)
+    e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## How to reset swipe the view automatically
+
+Swiped item can be reset by defining the `SfListView.SwipeOffSet` argument of `SfListView.SwipeEnded` event to `0` when the swiping action is completed.
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void ListView_SwipeEnded(object sender, SwipeEndedEventArgs e)
+{
+  if (e.Offset > 70)
```

Comment:
check whether  content is matched with code snippet

Reply:
resolved